package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "clean": "yarn workspaces foreach -p --include '@commercetools-website/*' run clean",
-    "start": "yarn workspace @commercetools-website/docs-smoke-test start",
+    "start": "yarn workspace @commercetools-website/documentation start",
```

Comment:
Why would you want to start all four sites at once?  I could understand building but starting?